Tribler/Main/Build/Win/tribler.nsi

```diff
@@ -110,179 +109,178 @@ LangString DESC_SecDefaultMagnet ${LANG_ENGLISH} "Associate magnet links with ${
+    File Tribler\Core\DecentralizedTracking\pymdht\core\bootstrap_stable
+    File Tribler\Core\DecentralizedTracking\pymdht\core\bootstrap_unstable
+
+    ; Install MSVCR 2008, 2010, 2012
+    SetOutPath "$INSTDIR"
+
+    ; Libraries dependant on 2008 are: Python, APSW
+    File vc_redist_90.exe
+    ExecWait "$INSTDIR\vc_redist_90.exe /q /norestart"
+
+    ; Libraries dependant on 2010 are:M2Crypto, netifaces
+    File vc_redist_100.exe
+    ExecWait "$INSTDIR\vc_redist_100.exe /q /norestart"
+
+    ; Libraries dependant on 2012 are: LevelDB, LibTorrent
+    File vc_redist_110.exe
+    ExecWait "$INSTDIR\vc_redist_110.exe /q /norestart"
+
+    FileOpen $9 "$INSTDIR\tribler.exe.log" w
+    FileWrite $9 ""
+    FileClose $9
+    AccessControl::GrantOnFile "$INSTDIR\tribler.exe.log" "(BU)" "FullAccess"
+    AccessControl::GrantOnFile "$INSTDIR\tribler.exe.log" "(S-1-5-32-545)" "FullAccess"
+
+    ; End
+    SetOutPath "$INSTDIR"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "DisplayName" "${PRODUCT}"
+    WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "NoModify" 1
+    WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "NoRepair" 1
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "UninstallString" "$INSTDIR\Uninstall.exe"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "InstallLocation" "$INSTDIR"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "DisplayVersion" '${VERSION}'
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "DisplayIcon" "$INSTDIR\${PRODUCT}.exe,0"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "Publisher" "The Tribler Team"
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "HelpLink" 'http://forum.tribler.org'
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "URLInfoAbout" 'http://www.tribler.org'
+    WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "URLUpdateInfo" 'http://www.tribler.org/trac/wiki/Download'
+    ${GetSize} "$INSTDIR" "/S=0K" $0 $1 $2
+    IntFmt $0 "0x%08X" $0
+    WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}" "EstimatedSize" "$0"
+
+    ; Now writing to KHEY_LOCAL_MACHINE only -- remove references to uninstall from current user
+    DeleteRegKey HKEY_CURRENT_USER "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\${PRODUCT}"
+    ; Remove old error log if present
+    Delete "$INSTDIR\tribler.exe.log"
+
+    WriteUninstaller "$INSTDIR\Uninstall.exe"
+
+    ; Add an application to the firewall exception list - All Networks - All IP Version - Enabled
+    SimpleFC::AddApplication "Tribler" "$INSTDIR\${PRODUCT}.exe" 0 2 "" 1
+    ; Pop $0 ; return error(1)/success(0)
```

Comment:
If we are not using this code, maybe we should just remove it? I saw some more occurrences of commented-out code.